src/utils/browserHistory.ts

```diff
@@ -0,0 +1,5 @@
+import { createBrowserHistory } from 'history';
```

Comment:
I'm a bit confused by this file, does it just re-export `createBrowserHistory` ?

Reply:
No, it doesn't simply re-exports `createBrowserHistory`; It invokes it and then exports the reference to the returned object of the function call. Since imported modules are cached internally, this means multiple files can have access to the same -- in this case, `history` object. Which is what was needed for the page redirects.

I think this is actually very widely used like that.